elements/pl-file-upload/pl-file-upload.py

```diff
@@ -103,6 +102,7 @@ def parse(element_html, data):
         parsed_files = json.loads(files)
     except ValueError:
```

Comment:
Probably want to catch if anything happens here (even though it will be a `ValueError` most of the time).
```suggestion
    except Exception:
```